Chinese/推荐美剧/Breaking.Bad.md

```diff
@@ -0,0 +1,2 @@
+[Breaking Bad](https://en.wikipedia.org/wiki/Breaking_Bad), 又名《绝命毒师》，剧情主要内容是讲述一位在某中学教书的天才化学家得了肺癌，并且仅有数月的存活时间，偶然的机会他接触到制造毒品(drug, meth，冰毒)可以大赚一笔钱。于是，他打算运用自己
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.